pydantic/fields.py

```diff
@@ -97,8 +99,9 @@ def __init__(self, default: Any, **kwargs: Any) -> None:
 
 
 def Field(
-    default: Any,
+    default: Any = Undefined,
     *,
+    default_factory: Optional['Callable[[], Any]'] = None,
```

Comment:
Had to use `'Callable...'` because it would raise
`TypeError: 'ABCMeta' object is not subscriptable`

Reply:
How is that possible? It already used in this module without an issues. That could be the case for collections.Callable, but it's a different thing.

Reply:
Ah right ! I didnt' see the `Callable` came from `.typing`. Thanks !